index.html

```diff
@@ -472,7 +472,7 @@ <h3>Role Mapping Table</h3>
 					</td>
 				</tr>
 				<tr id="role-map-button">
-					<th><a class="role-reference" href="#button"><code>button</code></a> with default values for <a class="property-reference" href="#aria-pressed"><code>aria-pressed</code></a> and <a class="property-reference" href="#aria-haspopup"><code>aria-haspopup</code></a></th>
+					<th><a class="role-reference" href="#button"><code>button</code></a> with default value for <a class="property-reference" href="#aria-pressed"><code>aria-pressed</code></a> and a value other than <code>true</code> or <code>menu</code> for <a class="property-reference" href="#aria-haspopup"><code>aria-haspopup</code></a></th>
```

Comment:
I verified that the default for pressed is "undefined", but wouldn't it be more clear and explicit to not rely on a separate definition of default value?

```suggestion
					<th><a class="role-reference" href="#button"><code>button</code></a> with undefined value for <a class="property-reference" href="#aria-pressed"><code>aria-pressed</code></a> and a value other than <code>true</code> or <code>menu</code> for <a class="property-reference" href="#aria-haspopup"><code>aria-haspopup</code></a></th>
```

Reply:
Agree completely that "undefined" is clearer than "default", and it matches the wording of the mapping for toggle button:
> button with defined value for aria-pressed